.github/workflows/clp-lint.yaml

```
@@ -19,6 +19,10 @@ jobs:
         with:
           python-version: "3.11"
 
+      - uses: "meteorengineer/setup-meteor@v1"
+        with:
+          meteor-release: "2.15"
```

```suggestion
      - id: parse_meteor_version
        run: |-
          meteor_version=$(grep --only-matching --perl-regexp '[\d.]+' .meteor/release)
          echo "meteor_version=${meteor_version}" >> $GITHUB_OUTPUT

      - uses: "meteorengineer/setup-meteor@v1"
        with:
          meteor-release: "${{steps.parse_meteor_version.outputs.meteor_version}}"
```

Untested, but you get the idea.

---

Yup, it works.

Instead of the `>>`, `| tee --append` can help us to verify from the GitHub workflow output that whether the version is parsed correctly.